Ansible/group_vars/all.sample

```diff
@@ -87,27 +116,30 @@ def_env_uuid: "unknown"
 
 # Environment database settings
 def_env_db_name: "trillian_envs"
-def_env_db_ip: "<IP_ADDRESS_HERE>"
+def_env_db_ip: "172.31.0.99"
 
 
 # Environment summary
 def_env_zonetype: "Advanced"
-# BEING DEPRECATED:
-def_env_use_sec_groups: yes
-def_env_prihost: "<IP_ADDRESS_HERE>"
-def_env_sechost: "<IP_ADDRESS_HERE>"
+def_env_use_sec_groups: yes # BEING DEPRECATED:
+
+# IP of the primary storage pool
+def_env_prihost: "172.31.0.101"
+
+# IP of the primary storage pool
+def_env_sechost: "172.31.0.101"
 def_env_pripath: "/acs/primary/"
 def_env_priprot: "NFS"
 def_env_secpath: "/acs/secondary/"
-# BEING DEPRECATED:
-def_env_zone_secgroups: "false"
+
+def_env_zone_secgroups: "false" # BEING DEPRECATED:
```

Comment:
@DaanHoogland  This is me basically reordering the comment. As per my discussion with Paul, he mentioned that these were being removed. However the placement of that comment made me think that everything after that was be depricated so I moved it to just that line that was being depricated.